ansible/pbTestScripts/vmDestroy.sh

```diff
@@ -94,8 +94,13 @@ listOS() {
 
 destroyVMs() {
 	local OS=$1
-	vagrant global-status --prune | awk "/adoptopenjdk$OS/ { print \$1 }" | xargs vagrant destroy -f
-	echo "Destroyed all $OS Vagrant VMs"
+	local ID=$(vagrant global-status --prune | awk "/adoptopenjdk$OS/ { print \$1 }")
+	if [[ $ID != "" ]]; then
```

Comment:
Ensure no syntax error when it's blank
```suggestion
	if [[ "$ID" != "" ]]; then
```

Reply:
Good spot :-)